vsts/scripts/pullAndTag.sh

```diff
@@ -124,8 +123,22 @@ if [ -n "$TESTINTEGRATIONCASEFILTER" ];then
 		# Always convert filter for runtime images to lower case
 		echo "Runtime image filter is set for $platformFilter with version $platformVersionFilter"
 
+		# Create a local file that consolidates the different runtime image files into one to be read from
+		# Note: we don't write this file to the drop folder as we don't want this file written to for every integration test job
+		sourceFile="$BUILD_SOURCESDIRECTORY/temp/images/runtime-images-acr.txt"
```

Comment:
is there any other way to elaborate why writing all the files in a single file?

Reply:
Each job of a pipeline is running in its own independent container, and we moved from having all of the runtime images being built within the scope of one job to having a job for each Debian flavor that we build a set of runtime images for. Within these jobs, we're building the runtime images associated with the given Debian flavor, as well as pushing those images to our dev ACR instance. When those jobs are done, and we move on to future steps of the pipeline, such as integration tests or pushing the images to our production ACR instance, we need to have a record of what images we built within the scope of this pipeline. These individual Debian flavor jobs will each write their own `.txt` file that is an inventory of all runtime images built and pushed to our dev ACR instance so that when we reach future jobs, we can consolidate all of those `.txt` files into one to have one complete list of all runtime images created during this pipeline that we can either test or bulk push to our production ACR instance.